pinot-tools/pom.xml

```diff
@@ -98,6 +103,21 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-math3</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
```

Comment:
@kishoreg @fx19880617 Doesn't this defeat the purpose on the reason that we took out the separate package for avro, orc, parquet readers and azure filesystem? How do you guys think?

Reply:
This is adding it only tools right not to core. We definitely need to extract out all things related to format and filesystems out of pinot-core and make it pluggable. But for now, this is the only solution I can think of adding support for different formats.